redhat/copr.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#Make sure that the tracking state is identical for master branch.
+if [[ "$(git rev-parse HEAD)" == "$(git log -1 --pretty=oneline origin/feature/redhatcopr | sed -E 's/^([^[:space:]]+).*/\1/')" ]]; then
```

Comment:
@snwh If merged, the string `origin/feature/redhatcopr` needs to be modified accordingly. For master branch: `origin/master`